src/Compilers/CSharp/Portable/Symbols/Wrapped/WrappedNamedTypeSymbol.cs

```diff
@@ -95,11 +95,19 @@ internal override bool MangleName
             }
         }
 
-        internal override string ExtensionName
+        internal override string ExtensionGroupingName
```

Comment:
>internal override string ExtensionGroupingName [](http://example.com/codeflow?start=8&length=46)

I think it will be better to not override these APIs in this class.